ansible_collections/arista/avd/molecule/eos_config_deploy_cvp/intended/structured_configs/cvp/cv_server.yml

```diff
@@ -1,95 +1,71 @@
 ---
 CVP_DEVICES:
-  DC1-BL1A:
-    name: DC1-BL1A
+  - fqdn: DC1-BL1A
```

Comment:
is this change from `name` to `fqdn` expected? it does not look like a fqdn here so would this mean that users now need to use fqdn somewhere?

Reply:
but `name` != `fqdn`.

Reply:
v3 useshttps://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_config_deploy_cvp/templates/cvp-devices-v3.j2

v1 uses:
 https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_config_deploy_cvp/templates/cvp-devices.j2
 
 so we should have `fqdn` with v3